sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/controller/AppController.java

```diff
@@ -44,6 +40,10 @@
     @Autowired
     private AppManagement appManagement;
 
+    @Value("${sentinel.dashboard.version}")
+    private String sertinelDashboardVersion;
```

Comment:
Here is a typo, `sertinel`->`sentinel`. Maybe dashboard can be omitted since here we know it's Sentinel Dashboard.